### ISSUE TITLE
Does not work with ESLint v2

### ISSUE THREAD

**Issue by gajus** [wix/eslint-plugin-lodash#22]
I am getting an error:

```
/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint-plugin-lodash/lib/rules/matches-shorthand.js:22
        return context.ecmaFeatures.objectLiteralShorthandProperties && astUtil.isEqEqEq(node) &&
                                   ^

TypeError: Cannot read property 'objectLiteralShorthandProperties' of undefined
    at canBeObjectLiteralWithShorthandProperty (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint-plugin-lodash/lib/rules/matches-shorthand.js:22:36)
    at isConjunctionOfEqEqEqToMemberOf (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint-plugin-lodash/lib/rules/matches-shorthand.js:29:35)
    at shouldPreferMatches (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint-plugin-lodash/lib/rules/matches-shorthand.js:48:16)
    at callExpressionReporters.always (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint-plugin-lodash/lib/rules/matches-shorthand.js:57:50)
    at EventEmitter.<anonymous> (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:184:13)
    at emitOne (events.js:95:20)
    at EventEmitter.emit (events.js:182:7)
    at NodeEventGenerator.enterNode (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint/lib/util/node-event-generator.js:42:22)
    at CodePathAnalyzer.enterNode (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint/lib/code-path-analysis/code-path-analyzer.js:592:23)
    at CommentEventGenerator.enterNode (/Users/gajus/Documents/dev/canonical-code-style/canonical/node_modules/eslint/lib/util/comment-event-generator.js:98:23)
```

**Comment by gajus** [comment]
> The ecmaFeatures property is now under a top-level parserOptions property.

http://eslint.org/docs/2.0.0/user-guide/migrating-to-2.0.0.html#language-options

**Comment by gajus** [comment]
Simply replacing `context.ecmaFeatures.objectLiteralShorthandProperties` with `context.parserOptions.ecmaFeatures.objectLiteralShorthandProperties` makes it work.

**Comment by ganimomer** [comment]
Fixed in v1.0.3
